src/score.js

```diff
@@ -127,7 +127,7 @@ export const makeCandidateScoreCalc = (config, hostname, chatDomain, apiSlug, ap
 
         // If privileged user asking candidate score of another user, get user site id from message
         // TODO: Allow Admins and Devs too, not just mods
-        if (isAskingForOtherUser && isModerator) {
+        if (isAskingForOtherUser && (isModerator || config.devIds.has(userId))) {
```

Comment:
To avoid accidentally creating problems during SO election, I only added `devIds` there - and even that we can revert